torch/nn/modules/activation.py

```diff
@@ -109,8 +109,8 @@ class Hardtanh(Module):
     The range of the linear region :math:`[-1, 1]` can be adjusted
 
     Args:
-        min_value: minimum value of the linear region range
-        max_value: maximum value of the linear region range
+        min_val: minimum value of the linear region range
+        max_val: maximum value of the linear region range
```

Comment:
Might be good to mention in the docstring that `max_value` and `min_value` kwargs should not be used?